app/connector/support/maven-plugin/pom.xml

```diff
@@ -29,6 +29,21 @@
   <name>Connector :: Support :: Maven Plugin</name>
   <packaging>maven-plugin</packaging>
 
+  <dependencyManagement>
```

Comment:
If no other POM is using these dependencies there's no need for declaring them in `<dependencyManagement>`, just put them as `<dependencies>` below. If other POMs will be using these declare them in the parent POM.